plugins/s3-viewer-common/package.json

```diff
@@ -31,13 +31,13 @@
     "postpack": "backstage-cli package postpack"
   },
   "devDependencies": {
-    "@backstage/cli": "^0.22.7"
+    "@backstage/cli": "^0.22.8"
   },
   "files": [
     "dist"
   ],
   "dependencies": {
     "@aws-sdk/client-s3": "^3.310.0",
```

Comment:
There's a bunch of places where `@aws-sdk` was not updated, please update to the same version everywhere.

The effect should be that the `yarn.lock` looks more reasonable and that duplicated dependencies like this should be removed: https://github.com/spreadshirt/backstage-plugin-s3/blob/1965d2c3d8c3b9ffc29195adce0f58b7350fed12/yarn.lock#L1366-L1380

Reply:
Aligned the dependencies with the versions used upstream. It looks quite promising now, since a lot of lines in the yarn.lock have been removed